eessi_container.sh

```diff
@@ -585,6 +585,11 @@ if [[ "${ACCESS}" == "ro" ]]; then
 fi
 
 if [[ "${ACCESS}" == "rw" ]]; then
+  if [[ ! -z ${JOB_CFG_FILE} ]]; then
+    # always launch build jobs triggered by the job with --fakeroot,
```

Comment:
clarify the comment, mention why we need `--fakeroot` (to also enable removal of things)?

Reply:
Done, also added a link to the issue